README.md

```diff
@@ -1,41 +1,11 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Yatch Booking App</b></h3>
```

Comment:
- [ ] I cannot create the database:

![image](https://github.com/akos29/book_appointment/assets/57421823/fdafe5b0-7e13-49c3-95f0-8784ada9190c)

![image](https://github.com/akos29/book_appointment/assets/57421823/f603c945-c0c6-4e95-85a9-1238286e8f58)

![image](https://github.com/akos29/book_appointment/assets/57421823/3f5bb90e-f242-4c9f-82df-14bccd3f9c4e)

This is because a lot from the rails installation has been deleted from the project repo:

![image](https://github.com/akos29/book_appointment/assets/57421823/1d65646b-2ec5-47e4-828d-3941f1f302be)

Some important folders like ./bin and ./lib are not in the project:

![image](https://github.com/akos29/book_appointment/assets/57421823/dcf7b1bc-bcfb-4439-8675-cd3cac476562)

Therefore, I can't test the project functionality or the tests.  Please restore the Rails installation to a functional state so it can be reviewed.  

The functionality, for the API, tests, models and controllers will be taken a look at in the next review. 👍